### DISCUSSION TITLE
Copilot account not activated

### DISCUSSION THREAD

**Discussion by zyxdSTU** [community/community#118754] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### the copilot page like this, seem not activated
![image](https://github.com/community/community/assets/26239665/f9999bd8-6b6e-45cd-aa14-cd6c231e3182)
### but I already paid
![image](https://github.com/community/community/assets/26239665/906fc7b7-803b-4a54-8e72-635843455fa5)
### and When I log in on the plugin like this
![image](https://github.com/community/community/assets/26239665/053ea815-efc6-4189-8b43-56854264fe9b)

**Comment by davecheney** [comment, marked as answer]
@zyxdSTU 
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
